knightos/commands/init.py

```diff
@@ -66,8 +66,18 @@ def init(ws, root, exists, site_packages, template, template_vars, vcs, force):
         install_local_kernel(ws.root, ws.kroot, template_vars['platform'], template_vars['kernel_path']);
 
     print("Installing packages...")
-    install.execute(template["install"])
     ws = Workspace(root)
```

Comment:
This needs to be moved back to IMMEDIATELY after install.execute, to refresh the list of installed packages